test/integration/targets/portmake/tasks/portmake.yml

```diff
@@ -5,7 +5,7 @@
   shell: pkg install -y subversion >/dev/null 2>&1
 
 - name: silently checkout a tested revision of the ports tree to make the tests more stable
-  svn checkout -r 493256 https://svn.FreeBSD.org/ports/head /usr/ports >/dev/null 2>&1
+  shell: svn checkout -r 493256 https://svn.FreeBSD.org/ports/head /usr/ports >/dev/null 2>&1
```

Comment:
While redirecting stdout to /dev/null may be a reasonable way to minimize output, we should avoid doing so for stderr unless the usual output there is excessive. Otherwise lack of stderr output on this (and other) tasks will make troubleshooting test failures very difficult.

Reply:
Also, you may want to use the subversion module here instead of shell:

https://docs.ansible.com/ansible/latest/modules/subversion_module.html